tests/plugins/clipboard.test.ts

```diff
@@ -436,12 +437,81 @@ describe("clipboard", () => {
     setCellContent(model, "B2", "abc");
     selectCell(model, "B2");
     copy(model, "B2");
-    expect(model.getters.getClipboardContent()).toBe("abc");
+    expect(model.getters.getClipboardTextContent()).toBe("abc");
 
     setCellContent(model, "B2", "= 1 + 2");
     selectCell(model, "B2");
     copy(model, "B2");
-    expect(model.getters.getClipboardContent()).toBe("3");
+    expect(model.getters.getClipboardTextContent()).toBe("3");
+  });
+
+  describe("Copied cells HTML", () => {
+    test("Copied group of cells are represented as an HTML table in the clipboard", async () => {
+      const model = new Model();
+      setCellContent(model, "A1", "1");
+      setCellContent(model, "B1", "2");
+      setCellContent(model, "A2", "3");
+      copy(model, "A1:B2");
+      const htmlContent = model.getters.getClipboardContent()[ClipboardContentType.HTML]!;
+      const parser = new DOMParser();
+      const parsedHTML = parser.parseFromString(htmlContent, "text/html");
```

Comment:
I would actually *not* parse it.
I think it would be more readable if you compare the html string. Maybe with `toMatchInlineSnapshot`
maybe with `import format from "xml-formatter";` to have something better formatted

Reply:
My point with parsing the clipboard content was to make sure that the HTML il actually valid. Since we generate the HTMl by hand, it would be really easy to mess something up (missing closing tag, `style:` instead of `style=`, ...)

But I can change if you feel that the readability of the tests are more important :)

Reply:
You've got a point.
We could have both :wink: 
Compare the raw string for readability and parse it and assert there's no error

Reply:
I kept parsing for a test, and tested directly the html string for the others. The test do a bit more than testing that we can parse as html, because the DOMParser will manage to parse about anything as html, and "fix" the errors as it goes.